nn_sdwan/tox.ini

```
@@ -0,0 +1,23 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py39
+envlist =
+    py{27,38,39}
```

```suggestion
    py{38,39}
```
Do we need to test on py27? The py27 tests failed due to presence of python 3 syntax. Also, should the base python be py38 or are you leaving it as py39 for when the agent bumps the python version?

---

No, I don't think we need to test on py27.

Originally we were developing this in a py39 environment, but then were told that the agent doesn't support py39 so now we're using a py38 environment. I believe the integration works the same in both, though. What do you suggest we change `basepython` and/or `envlist` to?

---

Got it. Let's use `basepython = py38` and `envlist` py38 

```suggestion
    py38
```